src/test/java/com/oscarcommerce/fw/ItemListContainerHelper.java

```diff
@@ -0,0 +1,25 @@
+package com.oscarcommerce.fw;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class ItemListContainerHelper extends BaseHelper {
+
+    public static final String BOOK_THE_SHELLCODERS_HANDBOOK_XPATH = "//a[@href='/en-gb/catalogue/the-shellcoders-handbook_209/']";
```

Comment:
Maybe worth to read this value from some file? Or pass all the way from features? Because it's tightly coupled to the data.